src/view/trip-events-list.js

```diff
@@ -0,0 +1 @@
+export const createTripEventsListTemplate = () => '<ul class="trip-events__list"></ul>';
```

Comment:
```suggestion
export const createTripEventsListTemplate = () => `<ul class="trip-events__list"></ul>`;
```
Пожалуйста, используй back-tick кавычки. Это требование к стилю в академии,... было во всяком случае. А еще удобно привыкнуть к этим кавычкам, удобно добавлять текст, ведь можно переносить на следующую строку или вставить кусочек вычислений. Ну и мода, опять же :)

Reply:
Этого не позволяет мне сделать eslint ссылаясь на правило 'Strings must use singlequote.'. Видимо строка слишком короткая чтобы оборачивать ее в back-ticks